### DISCUSSION TITLE
A nuget update any time soon?

### DISCUSSION THREAD

**Discussion by themronion** [xamarin/XamarinCommunityToolkit#939] [Q&A] [answered]
Hey! I am preparing an app for release and i currently have there some bugs that were fixed in the last commits. So i am wondering whether an update is planning in the nearest future?

**Comment by jfversluis** [comment, marked as answer]
I'm planning to release 1.0.3 early next week, maybe this weekend and 1.1 soon too :)

We're thinking about a more structured release cadence

**Reply by themronion** [reply]
Great! Thanks for the response!
